files/image_config/monit/container_checker

```diff
@@ -20,9 +20,34 @@ import docker
 import sys
 
 import swsssdk
-from sonic_py_common import multi_asic, device_info
+from sonic_py_common import multi_asic, device_info, daemon_base
 from swsscommon import swsscommon
 
+def get_asic_presence_list():
+    """
+    @summary: This function will get the asic presence list. On Supervisor, the list includes only the asics
+              for inserted and detected fabric cards. For non-supervisor cards, e.g. line card, the list should
+              contain all supported asics by the card. The function gets the asic list from CHASSIS_ASIC_TABLE from
+              CHASSIS_STATE_DB. The function assumes that the first N asic ids (asic0 to asic(N-1)) in
```

Comment:
can we move this API inside :  ./src/sonic-py-common/sonic_py_common/multi_asic.py so that it become more of library/generic function

Reply:
fixed.